src/components/Plugins/index.jsx

```diff
@@ -0,0 +1,39 @@
+import { Chart } from 'react-chartjs-2';
```

Comment:
Please rename `src/components/Plugins/index.jsx` to `src/components/ChartJsWrapper/registerTooltipPlugin` since this module is by nature strongly coupled with ChartJsWrapper.

Reply:
@asingh217 after reading the code and your comment earlier I realized that a proper way of handling this is by using a higher order component. Even though we could get by with strongly coupling Benchmarks with this code for this specific project, I want to create a solution we could easily reuse on other projects (e.g. firefox-health-dashboard). It also helps with code readability.

Here's an example of a higher order component:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/hocs/withErrorBoundary.jsx
You can see how it gets used in here:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/331a5a03fa96ecaff18865be6e391c3685cf16f0/src/containers/BugzillaBurndown/index.jsx#L53

Given that, you can probably use the hoc with the `App` component. If it does not work use it with the `Benchmark` component.

You can probably name it `withChartJsRegisteredGlobalPlugin`.

Let me know if you get into trouble or if what I say does not make sense.

Reply:
I will try creating a component for the plugin.

this statement is not clear "Given that, you can probably use the hoc with the App". Did you mean that i should add the call to withChartJsRegisteredGlobalPlugin in App/index.jsx

Reply:
You got it. Give it a try and see if it works on that component.

I will be off until Tuesday. Have a good weekend.